hyperspy/gui/egerton_quantification.py

```diff
@@ -145,7 +145,8 @@ def bg_to_plot(self, axes_manager=None, fill_with=np.nan):
         return to_return
 
     def span_selector_changed(self):
-        if (self.ss_left_value is np.nan) or (self.ss_right_value is np.nan):
+        if self.ss_left_value is np.nan or self.ss_right_value is np.nan or\
+                self.ss_right_value <= self.ss_left_value:
```

Comment:
To clarify: If this was an issue triggered by the widget, this would be the wrong place. If it was because of something else it is appropriate.